app/src/main/java/ch/epfl/sdp/blindly/main_screen/my_matches/chat/ChatActivity.kt

```diff
@@ -60,6 +65,17 @@ class ChatActivity : AppCompatActivity() {
         findViewById<RecyclerView>(R.id.recyclerView).adapter =
             getMessages()?.let { ChatAdapter(currentUserId, it) }
 
+        val matchName = intent.extras?.getString(USERNAME)
+
+        if (matchName != null) {
+            val nameBar = findViewById<Toolbar>(R.id.matchNameBar)
+            nameBar.title = intent.extras?.getString(USERNAME)
+            nameBar.setOnClickListener {
+                launchMatchProfileActivity()
```

Comment:
Nice! I think it's very intuitive to go to profile via nameBar.

Reply:
Yes I think it's nice too. If we feel like adding buttons and/or images can be better (as we said in meetings), we can do it later.